ec/src/scalar_mul/fixed_base.rs

```diff
@@ -9,19 +9,16 @@ use super::ScalarMul;
 pub struct FixedBase;
 
 impl FixedBase {
-    pub fn get_mul_window_size(num_scalars: usize) -> usize {
+    fn get_mul_window_size(num_scalars: usize) -> usize {
         if num_scalars < 32 {
             3
         } else {
             super::ln_without_floats(num_scalars)
         }
     }
 
-    pub fn get_window_table<T: ScalarMul>(
-        scalar_size: usize,
-        window: usize,
-        g: T,
-    ) -> Vec<Vec<T::MulBase>> {
+    pub fn get_window_table<T: ScalarMul>(window: usize, g: T) -> Vec<Vec<T::MulBase>> {
```

Comment:
I don't think we should get rid of `scalar_size`, because there might be some cases where this is smaller than `T::ScalarField::MODULUS_BIT_SIZE`, and constructing the larger table is wasted effort.